cmd/limactl/main.go

```diff
@@ -72,6 +73,24 @@ func newApp() *cobra.Command {
 			}
 			logrus.SetLevel(lvl)
 		}
+
+		logFormat, _ := cmd.Flags().GetString("log-format")
+		switch logFormat {
+		case "json":
+			formatter := new(logrus.JSONFormatter)
+			logrus.StandardLogger().SetFormatter(formatter)
+		case "text":
```

Comment:
Can we change this to `default`. `text` sounds confusing because **all** logging output will be text.

FWIW, I'm not too happy with `default` either, but at least it makes it obvious that it is the same as not specifying a log format.

Better suggestions welcome!

Reply:
"default" sounds good, waiting for for more feedback.

Reply:
I think "text" is fine, as it corresponds to logrus's terminology

Reply:
"default" should be an alias to another identifier

Reply:
What about `plain`?

https://docs.docker.com/reference/cli/docker/buildx/build/#progress

Reply:
> I think "text" is fine, as it corresponds to logrus's terminology

`text` allows a bunch of customization, like dropping timestamps etc, and I thought we might want to define shortcuts for some of them, and they would all use the TextFormatter.

But on second thoughts I think that might be overkill, and just supporting `text` and `json` with the default options may be all we need. Users can always transform JSON to whatever they want.

So I'm going to withdraw my objection to using `text`.

I don't think we need a `default` alias as long as we only have `text` and `json`.

Reply:
I agree we don't need alias.

Some users may want to modify the "test" logs, but I does not seem nice to add many --log-xxx options. It could be better if this came from a default configuration file.